enterprise/server/backends/distributed/distributed.go

```diff
@@ -611,6 +611,7 @@ func (c *Cache) Metadata(ctx context.Context, r *rspb.ResourceName) (*interfaces
 			c.log.Debugf("Metadata(%q) not found on peer %s", cacheproxy.ResourceIsolationString(r), peer)
 			continue
 		}
+		c.log.Debugf("Metadata(%q) lookup failed on peer %s: (err: %+v)", cacheproxy.ResourceIsolationString(r), peer, err)
```

Comment:
nit: for errors we typically use %v as there's generally no need to know the structure of the error message

Reply:
ok! done